### PR TITLE
[Merged by Bors] - feat: port Control.Fix

### PR DESCRIPTION
**Pull request by Shreyas4991**
Mathlib Commit : 207cfac9fcd06138865b5d04f7091e46d9320432

### CODE REVIEW

File: Mathlib/Control/Fix.lean [hunk 1]
```diff
@@ -67,22 +67,22 @@ it satisfies the equations:
   1. `fix f = f (fix f)`          (is a fixed point)
   2. `∀ X, f X ≤ X → fix f ≤ X`   (least fixed point)
 -/
-protected def fix (x : α) : Part <| β x :=
+protected noncomputable def fix (x : α) : Part <| β x :=
```
Comment (arienmalec) [inline Mathlib/Control/Fix.lean:72]: Add porting notes that `noncomputable` was required.

File: Mathlib/Control/Fix.lean [hunk 2]
```diff
@@ -114,7 +115,7 @@ end Part
 
 namespace Part
 
-instance : HasFix (Part α) :=
+noncomputable instance : HasFix (Part α) :=
```
Comment (arienmalec) [inline Mathlib/Control/Fix.lean]: porting notes on `noncomputable`

File: Mathlib/Control/Fix.lean [hunk 3]
```diff
@@ -123,9 +124,8 @@ open Sigma
 
 namespace Pi
 
-instance Part.hasFix {β} : HasFix (α → Part β) :=
+noncomputable instance Part.hasFix {β} : HasFix (α → Part β) :=
```
Comment (arienmalec) [inline Mathlib/Control/Fix.lean]: porting notes on `noncomputable`

**Review by arienmalec — APPROVED** [review]

**Comment by Shreyas4991** [comment]
Is there anything further to be done on this PR from my end?

**Comment by rwbarton** [comment]
bors merge

**Pull request closed without merging**